prdoc/pr_6293.prdoc

```diff
@@ -0,0 +1,10 @@
+title: Migrate pallet-nis benchmark to v2
```

Comment:
This PR should have the label silent and need not have a PRdoc.

Adding this label is something that only our contributors can do atm.